services/maven-metadata/maven-metadata.service.js

```diff
@@ -1,9 +1,14 @@
 'use strict'
 
 const Joi = require('@hapi/joi')
+const { optionalUrl } = require('../validators')
 const { renderVersionBadge } = require('../version')
 const { BaseXmlService } = require('..')
 
+const queryParamSchema = Joi.object({
+  metadataUrl: optionalUrl.required(),
```

Comment:
This could probably just be `url`. Thoughts?

Reply:
SGTM